data/integration/integration-packages.json

```
@@ -0,0 +1,172 @@
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/samsonasik/force-https-module",
+    "keywords": ["https", "middleware", "redirect"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/articus/path-handler",
+    "keywords": ["routing", "middleware", "api"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/mostafasy/mezzio-debugbar",
+    "keywords": ["debugbar", "middleware"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/shlinkio/shlink",
+    "keywords": ["url", "rest", "api"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/kynx/mezzio-openapi",
+    "keywords": ["openapi", "api", "mezzio"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/league/openapi-psr7-validator",
+    "keywords": ["http", "openapi", "validation"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/middlewares/http-authentication",
+    "keywords": ["http", "authentication", "middleware"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/middlewares/client-ip",
+    "keywords": ["http", "ip", "middleware"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/middlewares/negotiation",
+    "keywords": ["content", "encoding", "middleware"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/middlewares/whoops",
+    "keywords": ["error-handling", "middleware"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/middlewares/fast-route",
+    "keywords": ["router", "middleware"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/settermjd/mezzio-simple-user-manager",
+    "keywords": ["mezzio", "user", "authentication"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/settermjd/mezzio-eventmanager-integration",
+    "keywords": ["mezzio", "PSR-15", "PSR-7"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/settermjd/laminas-twilio-phone-number-validator",
+    "keywords": ["validator", "twilio", "laminas"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/mimmi20/mezzio-navigation",
+    "keywords": ["mezzio", "navigation"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/mimmi20/laminasviewrenderer-vite-url",
+    "keywords": ["laminas-view", "vite", "url"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/mimmi20/laminasviewrenderer-js-log",
+    "keywords": ["laminas-view", "js", "logger"],
+    "homepage": ""
+  },
+  {
+    "packagistUrl": "https://packagist.org/packages/mimmi20/mezzio-setlocale-middleware",
+    "keywords": ["mezzio", "middleware", "language"],
+    "homepage": ""
+  }
+]
```

I'd like to add one of my package too. It is a middleware to support file upload in GraphQL via [webonyx/graphql-php](https://github.com/webonyx/graphql-php):

```suggestion
 {
    "packagistUrl": "https://packagist.org/packages/ecodev/graphql-upload",
    "keywords": ["graphql", "upload", "middleware"],
    "homepage": ""
  },
]
```